android/libraries/rib-compiler-app/src/main/kotlin/com/uber/rib/compiler/RibInteractorProcessorPipeline.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2017. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.rib.compiler
+
+import com.uber.rib.core.RibInteractor
+import java.io.IOException
+import java.util.ArrayList
+import javax.lang.model.element.TypeElement
+
+/** The processor pipeline for [RibInteractor]  */
+open class RibInteractorProcessorPipeline(
+  processContext: ProcessContext,
+  private var interactorGenerator: Generator<InteractorAnnotatedClass>?
+) : TypeProcessorPipeline(processContext) {
+  private val annotationVerifier: AnnotationVerifier<InteractorAnnotatedClass>
+  private val builderAnnotatedClassesList: MutableList<InteractorAnnotatedClass?> = ArrayList()
+
+  /** @return the annotation type this processor pipeline deals with.
+   */
```

Comment:
nit - formatting of this kdoc

Reply:
good catch, ktfmt will fix this for us when we can enable it via spotless